### PR TITLE
Use Odsp Test Driver in Service Load Tests

### PR DESCRIPTION
**Pull request by anthony-murphy**

### CODE REVIEW

File: packages/test/service-load-test/src/nodeStressTest.ts [hunk 1]
```diff
@@ -207,55 +170,29 @@ async function runnerProcess(
...
-    const driveId = await getDriveId(loginInfo.server, "", undefined, { accessToken: odspTokens.accessToken });
+    const testDriver = createTestDriver(loginInfo);
 
     // Create a new file if a url wasn't provided
-    const url = args.url ?? await initialize(driveId, loginInfo);
+    const url = args.testId ?? await initialize(testDriver);
 
     const estRunningTimeMin = Math.floor(2 * profile.totalSendCount / (profile.opRatePerMin * profile.numClients));
-    console.log(`Connecting to ${args.url ? "existing" : "new"} Container targeting dataStore with URL:\n${url}`);
+    console.log(`Connecting to ${args.testId ? "existing" : "new"} Container targeting dataStore with URL:\n${url}`);
```
Comment (markfields) [inline packages/test/service-load-test/src/nodeStressTest.ts:195]: FYI this string

File: packages/test/service-load-test/src/testConfigFile.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export interface ITestConfig {
 export interface ITestTenant {
     server: string,
     username: string,
+    driveId: string,
```
Comment (markfields) [inline packages/test/service-load-test/src/testConfigFile.ts:16]: Why aren't we looking up the drive id at runtime anymore? It's redundant to have to specify it

**Review by markfields — COMMENTED** [review]
Love all the red! Just skimming on my phone in the middle of the night... will look again tinnitus to sign off.

File: packages/test/service-load-test/src/testConfigFile.ts (hunk 2 above)
Comment (anthony-murphy) [reply packages/test/service-load-test/src/testConfigFile.ts:16]: i was thinking the opposite. it never changes, so why look it up every time.

File: packages/test/service-load-test/src/nodeStressTest.ts (hunk 1 above)
Comment (anthony-murphy) [inline packages/test/service-load-test/src/nodeStressTest.ts:195]:
```suggestion
    console.log(`Connecting to ${args.testId ? "existing" : "new"} Container targeting with testId:\n${testId }`);
```

File: packages/test/service-load-test/src/testConfigFile.ts (hunk 2 above)
Comment (curtisman) [reply packages/test/service-load-test/src/testConfigFile.ts:16]: The problem is that now to set up, you will have to find out the driverId before you can target a tenant. 
The driveId isn't easy to find manually.

Comment (anthony-murphy) [reply packages/test/service-load-test/src/testConfigFile.ts:16]: my plan was to have a script to generate the config and add the drive id. the problem with discover here is it makes the service load test coupled to odsp which i'm trying to break

**Review by curtisman — APPROVED** [review]
:shipit:

File: packages/test/service-load-test/src/testConfigFile.ts (hunk 2 above)
Comment (curtisman) [reply packages/test/service-load-test/src/testConfigFile.ts:16]: The test driver should be able to do a rest call and figure it out though.

---
In reply to: [581434492](https://github.com/microsoft/FluidFramework/pull/5230#discussion_r581434492) [](ancestors = 581434492)

**Pull request merged**